src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -166,7 +166,7 @@ inline bool is_power_activation(const ov::Node* node) noexcept {
             return true;
         }
         return (1.0f != value);
-    } else if (auto power_op = std::dynamic_pointer_cast<ngraph::op::PowerIE>(node)) {
+    } else if (auto power_op = dynamic_cast<const ngraph::op::PowerIE*>(node)) {
```

Comment:
can we keep unchanged?

Reply:
> can we keep unchanged?

If keep unchanged, this api `std::shared_ptr<T> dynamic_pointer_cast(const ::ov::Any& any)` can not be deprecated.